WORKSPACE

```diff
@@ -66,18 +72,35 @@ vaticle_dependencies_tool_maven_artifacts = "maven_artifacts")
 vaticle_dependencies_ci_pip()
 
 # Load //builder/grpc
-load("@vaticle_dependencies//builder/grpc:deps.bzl", grpc_deps = "deps")
-grpc_deps()
+load("@vaticle_dependencies//builder/grpc:deps.bzl", vaticle_grpc_deps = "deps")
+vaticle_grpc_deps()
+
+load("@rules_proto_grpc//:repositories.bzl", "rules_proto_grpc_repos", "rules_proto_grpc_toolchains")
+rules_proto_grpc_toolchains()
+rules_proto_grpc_repos()
+
+load("@rules_proto//proto:repositories.bzl", "rules_proto_dependencies", "rules_proto_toolchains")
+rules_proto_dependencies()
+rules_proto_toolchains()
```

Comment:
instead of `stackb_rules_proto`, we now use `rules_proto_grpc` which provide rules to compile grpc in Java, Rust, and Python. However, TypeScript we still implement somewhat ourselves.